src/app/notifications/components/SearchFailure/index.tsx

```diff
@@ -18,10 +18,16 @@ interface Props {
 const SearchFailure = ({ dismiss }: Props) => {
   const window = assertWindow();
   const [isFadingOut, setIsFadingOut] = React.useState(false);
+  const [shouldAllowDismissal, setShouldAllowDismissal] = React.useState(false);
```

Comment:
I've added this because clicking on a search result on mobile triggers a scroll event which will make the message fade out immediately. Not sure if this is the best solution.

Reply:
i haven't looked at your code yet, but i would think that maybe adding another timeout to accomplish "the message must be shown for x amount of time before we will automatically dismiss it" would be a good solution